### ISSUE TITLE
indices = 2 is not in [0, 1)

### ISSUE THREAD

**Issue by rdulmina** [keras-team/keras#12022]
Hi im working on seq2sql project and I successfully build a model but when training I get the following error also I'm not using any Keras embedding layer. the bellow is my code.

[code](https://gist.github.com/rdulmina/870803999bd099581afeb60aebc1953d)

the shapes of the Equestion, Col_Embeddings, and y_train is (10, 12, 3, 100) ,(10, 13, 100) and (10, 12).i searched about this error but all that cases they have used an embedding layer incorrectly. But here though **I'm not using any embedding layer** i get this error
`indices = 2 is not in [0, 1)
[[{{node lambda_3/embedding_lookup_2}} = GatherV2[Taxis=DT_INT32, Tindices=DT_INT32, Tparams=DT_FLOAT, _class=["loc:@col_token_2"], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_col_token_2_0_1, lambda_3/Assign_2, lambda_3/embedding_lookup_2/axis)]]`

please help

**Comment by msymp** [comment]
Hello @rdulmina , please post this on the TensorFlow issues repo, since it more relevant there, since there is no Keras impact in the code. Hence we close it here. Thanks.
